src/main/scala/nl.knaw.dans.easy.deposit/servlets/package.scala

```diff
@@ -95,29 +101,57 @@ package object servlets extends DebugEnhancedLogging {
         // a __MACOSX gets deleted because its content was deleted
         // it will simply not be a directory anymore and not cause trouble
         if (file.isDirectory && (file.isEmpty || file.name == "__MACOSX")) {
-          logger.info(s"[$id] cleaning up $file")
+          logger.info(s"[$draftDeposit] cleaning up $file")
```

Comment:
`draftDeposit` is a `DepositDir`, not an identifier...

```suggestion
          logger.info(s"[${ draftDeposit.id }] cleaning up $file")
```

or maybe you could extract `draftDeposit.id` since you use it 4x